src/shared_library.h

```diff
@@ -65,6 +65,14 @@ class SharedLibrary {
   Status GetEntrypoint(
       void* handle, const std::string& name, const bool optional, void** befn);
 
+  // Add an additional dependency directory to PATH (Windows-only).
+  Status AddAdditionalDependencyDir(
+      const std::string& additional_path, std::wstring& original_path);
+
+  // Restore PATH to its original configuration. Should be used in
+  // conjunction with AddAdditionalDependencyDir (Windows-only).
```

Comment:
nit: remove `(Windows-only)`

Reply:
Even if these new functions are Windows-only atm?

Reply:
It is just a matter of taste. This interface is abstracting away the platform-specific which is why I don't think you need to call out Windows here. You can keep it to highlight the behavior difference but then you should detail on what is the behavior for non-windows platform.